mteb/evaluation/evaluators/InstructionRetrievalEvaluator.py

```diff
@@ -30,6 +30,6 @@ def __call__(
                 queries,
                 self.top_k,
                 self.score_function,
-                instructions=instructions,
+                prompt_name=instructions,
```

Comment:
The instruction is actually separate from the `prompt_name` and both should be passed.

Reply:
Perhaps the UI isn't showing it, but I think this still needs to be updated.